cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/Main.java

```diff
@@ -462,14 +472,23 @@ private String promptForInput(String msg) {
         @CommandLine.Option(names = { "--version", "-v" }, hidden = true)
         private boolean versionFlag;
 
+        @CommandLine.Parameters
+        private List<String> argList = new ArrayList<>();
```

Comment:
Why so we introduce this arglist?

Reply:
Mistakenly added this change here. Will revert this.